ops/Dockerfile.opnode

```diff
@@ -7,10 +7,16 @@ COPY go.mod /app/go.mod
 COPY go.sum /app/go.sum
 
 WORKDIR /app
+COPY ./ops/opnode-entrypoint.sh /app/opnode-entrypoint.sh
+COPY ./ops/rollup.json /app/rollup.json
+
 COPY ./opnode /app/opnode
 RUN make opnode
 
 FROM alpine:3.13
```

Comment:
Can you update this to `alpine:3.15` so that we use the same image that we use in the monorepo as well?